tests/manual-test-cases/Group21-Registries/21-1-Whitelist.robot

```diff
@@ -37,12 +37,14 @@ Setup Harbor
     Remove Environment Variable  BRIDGE_NETWORK
     Remove Environment Variable  PUBLIC_NETWORK
 
+    Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Pass Execution  Test skipped on ESXi as Harbor is only supported on VC
```

Comment:
I have re-written this test to work in Nightly Nimbus and it currently runs on ESXi server.  Is this line and the line below a merge conflict?